Makefile

```diff
@@ -25,6 +25,7 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
+TEST_TAGS ?= basic
```

Comment:
@hferentschik working on it now. Only I would change CUCUMBER_OPTS to GODOG_OPTS as except tags and feature paths, all other options are relevant only for godog, for example StopOnFailure, NoColors etc.